app/controllers/podcast_metrics_controller.rb

```diff
@@ -3,13 +3,32 @@ class PodcastMetricsController < ApplicationController
 
   before_action :set_podcast
   before_action :check_clickhouse, except: %i[show]
-  before_action :set_date_range, except: %i[dropdays]
-  before_action :set_uniques, only: %i[show uniques]
-  before_action :set_dropday_range, only: %i[show dropdays]
 
   def show
   end
 
+  def episode_sparkline
+    @episode = Episode.find_by(guid: metrics_params[:episode_id])
+    @prev_episode = Episode.find_by(guid: metrics_params[:prev_episode_id])
+
+    @alltime_downloads =
```

Comment:
Why are you querying for all-time?  I can't quite figure out how this impacts the sparklines.

Or are those little arrow icons not static?  I sort of assumed they were just the static "up or down".